recipe/meta.yaml

```diff
@@ -15,15 +15,27 @@ build:
 
 requirements:
     build:
+        - toolchain
```

Comment:
Please remove this. There are only a few flags from this that are really needed and I'd rather you copy and paste them in the `build.sh`. As this package changes and the reproducibility of the build is broken and I do not agree with that bad  practice.